scripts/includes/common.sh

```diff
@@ -140,8 +140,13 @@ configure_st2_user () {
   # Generate ssh keys on StackStorm box and copy over public key into remote box.
   # NOTE: If the file already exists and is non-empty, then assume the key does not need
   # to be generated again.
+  ELMAJVER=$(cat /etc/redhat-release | sed 's/[^0-9.]*\([0-9.]\).*/\1/')
   if ! sudo test -s ${SYSTEM_HOME}/.ssh/stanley_rsa; then
-    sudo ssh-keygen -f ${SYSTEM_HOME}/.ssh/stanley_rsa -P ""
+    if [[ "$ELMAJVER" == 8 ]]; then
+        # EL8 added -m PEM to force RSA PEM format
+        PEM="-m PEM"
```

Comment:
After looking at it once again, I don't think it's good idea to include special case with "PEM" `ssh-keygen` param for EL8 installer.

The reason is that `curl|bash` script by idea is a demo installer.
With that, default, easy, clean and the most popular straightforward settings would be ideal to use.
Additionally, consistency across the platforms in other installers needs to be preserved and deviation is not recommended.

I would assume hardening cases like that are perfectly acceptable for other installers, with more extended  production-friendly configuration settings and options. For curl|bash let's please keep it simple.

Reply:
pushing